### PR TITLE
feat: Added Wishlist Page

### PR DESCRIPTION
**Pull request by MaruthiPothuganti**
- Wishlist Management page with cards with actions-buttons of Move to cart and Remove from Wishlist.

### CODE REVIEW

File: Wishlist/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    gap: 1rem;
+}
+
+.card-btn {
+    width: 20rem;
+}
+
+.ecom-card {
+    width: 25rem;
+    /* height: 45rem; */
```
Comment (shuklahimanshu81) [inline Wishlist/wishlist.css]: Remove comment.

File: Wishlist/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+            </div>
+
+        </div>
+
+    </div>
+
+
+
+
+
```
Comment (shuklahimanshu81) [inline Wishlist/wishlist.html]: Remove unnecessary space.

**Review by shivampandey0 — COMMENTED** [review]
Clean and easy-to-understand code.

**Review by shuklahimanshu81 — APPROVED** [review]
Clean code structure. Everything is good, go for merge.

File: Wishlist/wishlist.css [hunk 3]
```diff
@@ -0,0 +1,31 @@
+@import url("/style.css");
+.card-container {
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
```
Comment (kotalwar-krishna) [inline Wishlist/wishlist.css]: for gap here you use px unit and next you are using rem, be consisistent while using units

File: Wishlist/wishlist.css [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+.cards {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 1rem;
```
Comment (kotalwar-krishna) [inline Wishlist/wishlist.css:11]: refer for above comment

File: Wishlist/wishlist.css [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    justify-content: space-around;
+}
+
+.action-btns {
+    display: flex;
```
Comment (lannisterrr) [inline Wishlist/wishlist.css:16]: Suggestion : this could be a seperate utility class.

File: Wishlist/wishlist.css (hunk 1 above)
Comment (kotalwar-krishna) [inline Wishlist/wishlist.css]: remove comment

**Review by lannisterrr — APPROVED** [review]

File: Wishlist/wishlist.html [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+                </button>
+                    <button class="card-btn btn-secondary">Remove from Wishlist <ion-icon name="arrow-forward-circle-outline"></ion-icon></button>
+                </div>
+
+            </div>
+
+        </div>
+
+    </div>
+
```
Comment (kotalwar-krishna) [inline Wishlist/wishlist.html:129]: remove spaces

**Review by kotalwar-krishna — APPROVED** [review]
good to merge

File: Wishlist/wishlist.css (hunk 4 above)
Comment (MaruthiPothuganti) [reply Wishlist/wishlist.css:11]: Thankyou for the suggestion. Will make the change

File: Wishlist/wishlist.css (hunk 5 above)
Comment (MaruthiPothuganti) [reply Wishlist/wishlist.css:16]: A utility class  is already defined for this use, for some reason, it was not working. So, i rewrote the code here

**Pull request merged**